src/Engine/GameAction.java

```diff
@@ -253,17 +252,11 @@ public GameEventQueue getEvents(MapMaster gameMap)
           ArrayList<Unit> ary = new ArrayList<Unit>();
           ary.add(obstacle);
           boolean fatal = true;
-          MassDamageEvent mde = new MassDamageEvent(who, ary, obstacle.getHP()+1, fatal);
-          UnitDieEvent ude = new UnitDieEvent(obstacle);
+          MassDamageEvent mde = new MassDamageEvent(who, ary, obstacle.getHP() + 1, fatal);
           buildEvents.add(mde);
-          buildEvents.add(ude);
```

Comment:
If we no longer have a `UnitDieEvent` because mass damage covers it, we should remove it from the comment up at 236.

Reply:
The death+defeat event(s) are added by `Utils.enqueueDeathEvent()`.
As far as I can tell, the comment is still correct.